gwdetchar/lasso/__main__.py

```diff
@@ -31,9 +31,9 @@
 
 from sklearn import linear_model
 from sklearn.preprocessing import scale
+from sklearn.preprocessing import *
```

Comment:
It's a bit unusual to import with `*`. Perhaps better to just import what you need. This may help solve some flake8 errors